### PR TITLE
docs(forms): Added FormControl initialization information

### PR DESCRIPTION
**Pull request by fank**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: Documentation
```
## **What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://github.com/angular/angular/issues/11940#issuecomment-249929708

### CODE REVIEW

**Comment by Fank** [comment]
I signed it!

**Comment by vicb** [comment]
Please merge the commits and make sure

> The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Comment by Fank** [comment]
@vicb sorry for misunderstanding, but i thought the title in the PR will be taken by merging it. So the commit names in a PR will be ignored. Fixed it.

File: modules/@angular/forms/src/model.ts [hunk 1]
```diff
@@ -625,6 +625,8 @@ export abstract class AbstractControl {
  *
  * You can also initialize the control with a form state object on instantiation,
  * which includes both the value and whether or not the control is disabled.
+ * You can't use the value key without the disabled key,
```
Comment (kara) [inline modules/@angular/forms/src/model.ts]: Nitpick: need a semi-colon here :)

`...without the disabled key; both are required...`

**Review by kara — CHANGES_REQUESTED** [review]

File: modules/@angular/forms/src/model.ts (hunk 1 above)
Comment (Fank) [inline modules/@angular/forms/src/model.ts]: Done

**Review by kara — APPROVED** [review]
LGTM

**Comment by Fank** [comment]
The failed CI isn't related to this change.

**Pull request merged**